packages/lesswrong/components/posts/AllPostsPageSettings.jsx

```diff
@@ -158,15 +150,15 @@ class AllPostsPageSettings extends Component {
           <MetaInfo className={classes.selectionTitle}>
             Sorted by:
           </MetaInfo>
-          {views.map(view => {
+          {Object.entries(views).map(view => {
```

Comment:
Better written as `views.map((view, index) => {`. That avoids having to identify the view and its index by confusing array indices.